server/src/main/java/org/elasticsearch/index/engine/ThreadPoolMergeExecutorService.java

```diff
@@ -87,8 +87,8 @@ public class ThreadPoolMergeExecutorService {
     private ThreadPoolMergeExecutorService(ThreadPool threadPool) {
         this.executorService = threadPool.executor(ThreadPool.Names.MERGE);
         this.maxConcurrentMerges = threadPool.info(ThreadPool.Names.MERGE).getMax();
-        this.concurrentMergesFloorLimitForThrottling = maxConcurrentMerges * 2;
-        this.concurrentMergesCeilLimitForThrottling = maxConcurrentMerges * 4;
+        this.concurrentMergesFloorLimitForThrottling = 2;
+        this.concurrentMergesCeilLimitForThrottling = maxConcurrentMerges * 2;
     }
```

Comment:
Perhaps assert that `concurrentMergesCeilLimitForThrottling >= concurrentMergesFloorLimitForThrottling` - I feel like it adds readabililty.